src/Sylius/Bundle/ApiBundle/Converter/IriToIdentifierConverter.php

```diff
@@ -73,6 +73,8 @@ public function isIdentifier($fieldValue): bool
             return false;
         }
 
+        $fieldValue = filter_var($fieldValue, FILTER_SANITIZE_URL);
```

Comment:
As far as I understand, it will only cleanup field data for IRI conversion and full `comment` (from example) will go into DB?

Sanitize option doesn't do validation and always will return what have been supplied (int, invalid url strings), but my only concern is support for non-latin paths.

> Sanitize the string by remove all characters except `latin letters` 

https://www.php.net/manual/en/filter.constants.php#constant.filter-sanitize-url

Reply:
> As far as I understand, it will only cleanup field data for IRI conversion and full comment (from example) will go into DB?
yes exactly.

> Sanitize option doesn't do validation and always will return what have been supplied (int, invalid url strings), but my only concern is support for non-latin paths.
Could you please give an example of what url's you mean ?

Reply:
@diimpp 
I added test coverage*